src/common/snippets/include/snippets/pass/filter_fused.hpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <transformations_visibility.hpp>
+#include <ngraph/pass/pass.hpp>
+#include <snippets/itt.hpp>
+
+namespace ngraph {
+namespace snippets {
+namespace pass {
+
+/**
+ * @interface FilterFused
+ * @brief Mark operations that will be fused on plugin side (but not yet in snippets) so they'll be ignored by snippets.
+ * @ingroup snippets
+ */
+class TRANSFORMATIONS_API FilterFused : public ngraph::pass::FunctionPass {
```

Comment:
As we discussed some time ago: this pass is fully plug-in (device) specific. Internal logic have to be passed from a plugin via callback. 
I would also suggest to name this pass in more generic way, From the plug-in perspective there might be different reasons not to include node in subgraph besides it will be fused. The same for corresponding RTInfo field.

Reply:
Ok, done